api/src/main/java/org/eclipse/microprofile/auth/LoginConfig.java

```diff
@@ -37,27 +37,19 @@
 @Retention(RetentionPolicy.RUNTIME)
 public @interface LoginConfig {
     /**
-     * Soften the requirements here:
-     *
      * The authMethod is used to configure the authentication mechanism for the
      * JAX-RS application. As a prerequisite to gaining access to any web resources
      * which are protected by an authorization constraint, a user must have
-     * authenticated using the configured mechanism. Supported values include
-     * "BASIC", "DIGEST", "FORM", "CLIENT-CERT", "MP-JWT", or a vendor-specific
```

Comment:
Why did you delete these?

Reply:
@Emily-Jiang The idea is stop start preparing for the eventual removal of this annotation; there was a long discussion in of the old issues with many people preferring it; but it can also be a very sensitve thing to do in some cases, so what we agreed with Romain is to start with dropping any references to all but `MP-JWT` authentication mechanism for the users not to keep thinking that the MP JWT API introduces the annotation which can be used even for non-MP-JWT flows (which was one of the problems discussed in the earlier discussion), the scope of the MP JWT should be all about MP JWT.

Reply:
ok, thanks